consul/prepared_query_endpoint.go

```diff
@@ -73,8 +80,8 @@ func (p *PreparedQuery) Apply(args *structs.PreparedQueryRequest, reply *string)
 		if query == nil {
 			return fmt.Errorf("Cannot modify non-existent prepared query: '%s'", args.Query.ID)
 		}
-		if (query.Token != args.Token) && (acl != nil && !acl.QueryModify()) {
-			p.srv.logger.Printf("[WARN] consul.prepared_query: Operation on prepared query '%s' denied because ACL didn't match ACL used to create the query, and a management token wasn't supplied", args.Query.ID)
+		if acl != nil && !acl.PreparedQueryWrite(query.GetACLPrefix()) {
+			p.srv.logger.Printf("[DEBUG] consul.prepared_query: Operation on prepared query '%s' denied due to ACLs", args.Query.ID)
```

Comment:
I think these should remain WARN-level. I think for the others we used DEBUG-level as we dropped entries during filtering.